Q/qpOASES/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qpOASES"
+version = v"3.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/coin-or/qpOASES/archive/refs/tags/releases/$(version).zip", "28dbe55fdfdca5f8a43e8fbfbdb5a90c52546568f8ecc7b29bd1d42dc1ccdd2a"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd qpOASES-releases-3.2.1/
+cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
```

Comment:
Just for readability
```suggestion
cmake -B build \
    -DCMAKE_INSTALL_PREFIX=$prefix \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release
```